gulpfile.babel.js

```diff
@@ -77,3 +77,12 @@ gulp.task('stop', finish => {
     }, 1000);
   });
 });
+
+
+gulp.task('platform-test', function (cb) {
```

Comment:
Also, we should use arrow function instead of old style function as follows:
OLD style:
  function(cb) {
     ...
  }

NEW style:
  callback => {
    ...
  }